src/main/java/io/strimzi/kafka/bridge/converter/DefaultSerializer.java

```diff
@@ -24,7 +24,7 @@ public void configure(Map<String, ?> configs, boolean isKey) {
     public byte[] serialize(String topic, T data) {
 
         if (data == null)
-            return null;
+            return new byte[0];
```

Comment:
Why?

Reply:
This is what spotbugs suggested. I can revert and suppress a warning in this method.

Reply:
what's the warning it's raising?

Reply:
```[ERROR] Should io.strimzi.kafka.bridge.converter.DefaultSerializer.serialize(String, Object) return a zero length array rather than null? [io.strimzi.kafka.bridge.converter.DefaultSerializer] At DefaultSerializer.java:[line 27] PZLA_PREFER_ZERO_LENGTH_ARRAYS
```

Reply:
I would suppress this warning tbh